### DISCUSSION TITLE
Other interfaces than OpenAPI

### DISCUSSION THREAD

**Discussion by tridentsx** [ApicurioArchive/apicurio-studio#2280] [General]
What is the ambition with apicurio studio ? Is it only OpenAPI AsyncAPI or is the ambition to support other schema driven interfaces such as gRPC or AVRO ?

**Comment by EricWittmann** [comment]
The ambition starting with version 1.0 (which is a bit of a rewrite/re-imagining) is to support additional types, specifically focused on these main 5:

* OpenAPI
* AsyncAPI
* Apache Avro
* Google Protocol Buffers
* JSON Schema
